README.md

```diff
@@ -1304,10 +1304,12 @@ class {'collectd::plugin::sensors':
 class {'collectd::plugin::snmp':
   data  =>  {
     amavis_incoming_messages => {
-      'Type'     => 'counter',
-      'Table'    => false,
-      'Instance' => 'amavis.inMsgs',
-      'Values'   => ['AMAVIS-MIB::inMsgs.0']
+      'Type'        => 'counter',
+      'Table'       => false,
+      'Instance'    => 'amavis.inMsgs',
+      'Values'      => ['AMAVIS-MIB::inMsgs.0']
+      'Ignore'      => '00:00'
```

Comment:
I honestly don't know how `Ignore` works in this context. Is `00:00` a good example?